gammu/asyncworker.py

```diff
@@ -6,9 +6,9 @@
 class GammuAsyncThread(gammu.worker.GammuThread):
     """Thread for phone communication."""
 
-    def __init__(self, queue, config, callback):
+    def __init__(self, queue, config, callback, pull_func):
```

Comment:
Shouldn't the new parameter have a default value?

Reply:
We provide all default parameters for GammuAsyncWorker, but for GammuAsyncThread there has never been defaults parameters, this is an internal implementation detail.